CPAC/QPP/prep_QPP.py

```diff
@@ -0,0 +1,548 @@
+def gather_nifti_globs(pipeline_output_folder,resource_list,derivatives=None):
+    import os
```

Comment:
What is different here compared to the original gather_nifti_globs in cpac_group_runner? Should combine functionality into the original one.